src/detail/connection_handle.cpp

```diff
@@ -35,6 +35,10 @@ namespace sqlpp
   {
     namespace detail
     {
+      #if LIBMYSQL_VERSION_ID > 80000
+        typedef bool my_bool;
```

Comment:
Please use `using my_bool = bool` instead to be in line with the rest of the code.

Reply:
Fixed. Also maybe I should move this into it's own header as this block is in 2 files, src/detail/connection_handle.cpp and src/detail/prepared_statement_handle.h.

Let me know what you would prefer.

Reply:
Yes, I think it should be in a separate header, like `version_compatibility.h` maybe.